Gemfile.lock

```diff
@@ -1064,4 +1080,4 @@ DEPENDENCIES
   zip-zip
 
 BUNDLED WITH
-   2.2.25
+   2.2.26
```

Comment:
We may have issues with prod builds if we change bundler version. If we have to change, better update to 2.4 latest so that we have a significant improvement, instead of doing a very minor update.

Reply:
I reverted this